desc/compute/_field.py

```diff
@@ -2753,3 +2782,60 @@ def _B_dot_gradB_z(params, transforms, profiles, data, **kwargs):
         + data["B^zeta"] * data["|B|_zz"]
     )
     return data
+
+
+@register_compute_fun(
+    name="kappa",
+    label="\\kappa",
+    units="m^{-1}",
+    units_long="Inverse meters",
+    description="Curvature vector of magnetic field lines",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["J", "|B|", "b", "grad(|B|)"],
+)
+def _kappa(params, transforms, profiles, data, **kwargs):
+    data["kappa"] = -(
+        cross(data["b"], mu_0 * data["J"] + cross(data["b"], data["grad(|B|)"])).T
+        / data["|B|"]
+    ).T
+    return data
+
+
+@register_compute_fun(
+    name="kappa_n",
+    label="\\kappa_n",
+    units="m^{-1}",
+    units_long="Inverse meters",
+    description="Normal curvature vector of magnetic field lines",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["kappa", "n"],
+)
+def _kappa_n(params, transforms, profiles, data, **kwargs):
+    data["kappa_n"] = dot(data["kappa"], data["n"])
+    return data
+
+
+@register_compute_fun(
+    name="kappa_g",
+    label="\\kappa_g",
```

Comment:
Similar comment as above. Not sure that people will know `\\kappa_g` means "geodesic curvature". But not a big deal either way (it only really affects plotting).

Reply:
see prev comment